link/src/main/java/com/stripe/android/link/injection/LinkInjectable.kt

```diff
@@ -0,0 +1,24 @@
+package com.stripe.android.link.injection
+
+import com.stripe.android.core.injection.Injectable
+import com.stripe.android.core.injection.Injector
+
+/**
+ * Mark a class that can be injected by a [Injector] and does not support fallback.
+ * It's implemented by ViewModel Factories when the ViewModel lifecycle is shorter than the activity
+ * that contains it. In those cases, they receive the [Injector] directly as a constructor parameter
+ * and should not be responsible for recreating the dependency graph.
+ *
+ * @see [Injectable]
+ */
+internal interface LinkInjectable : Injectable<Unit> {
```

Comment:
Same here, could you rename it to `NonFallbackInjectable` so that we could move them to stripe-core later

Reply:
Done